funnel/views/participant.py

```diff
@@ -127,37 +128,34 @@ def participant_edit(profile, project, participant):
     return render_form(form=form, title=_(u"Edit Participant"), submit=_(u"Save changes"))
 
 
-@app.route('/<profile>/<project>/participant', methods=['GET', 'POST'])
-@funnelapp.route('/<project>/participant', methods=['GET', 'POST'], subdomain='<profile>')
+@app.route('/participant', methods=['GET', 'POST'])
+@funnelapp.route('/participant', methods=['GET', 'POST'], subdomain='<profile>')
```

Comment:
This too needs to be under `/contacts`.

Reply:
Can this be `/contacts/participant`
Can we now remove the GET method of this endpoint?

Reply:
Add contacts view and updated the endpoint to `/contacts/connect`